scripts/run_backend_tests.py

```diff
@@ -659,6 +695,25 @@ def check_coverage(
     process = subprocess.run(
         cmd, capture_output=True, encoding='utf-8', env=env,
         check=False)
+
+    not_covered_lines = []
+    is_first_line = True
+    output_lines = process.stdout.split('\n')
+
+    for index, line in enumerate(output_lines):
+        if line and (' 100%' not in line and '-----' not in line
+         and 'Name' not in line):
+            if is_first_line and index > 0:
+                not_covered_lines.append(output_lines[0])
+                not_covered_lines.append(output_lines[1])
+                is_first_line = False
+            not_covered_lines.append(line)
+            i = index
```

Comment:
Don't know what i is, please improve the naming to explain what concept you are representing here.

Reply:
@seanlip i removed i in updated code.